ui/components/multichain/pages/send/components/account-picker.test.tsx

```diff
@@ -7,6 +7,33 @@ import { SendPageAccountPicker } from '.';
 const render = (props = {}) => {
   const store = configureStore({
     ...mockState,
+   metamask: {...mockState.metamask,
```

Comment:
Does this pass lint?

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/23006/commits/58f909010573d635f64dc02d58508fb316309d97